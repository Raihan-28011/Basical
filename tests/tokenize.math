(2 + 3) * 3.4 / 8 % (2 ** 3)
3239 + 32.343 ** (323 * 6565.5656) % 439894 / 98383
sqrt(44.434) + 4393
