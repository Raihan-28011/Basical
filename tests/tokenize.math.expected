( 2 t_plus 3 ) t_pow 3.4 t_slash 8 t_mod ( 2 t_pow 3 ) 
3239 t_plus 32.343 t_pow ( 323 t_pow 6565.5656 ) t_mod 439894 t_slash 98383 
sqrt ( 44.434 ) t_plus 4393 
 t_eof
